ui/css/itcss/components/send.scss

```diff
@@ -585,7 +585,6 @@
   }
 
   &__asset-dropdown {
-    height: 54px;
```

Comment:
I'm a fan of this fix. Thanks for proposing this!

One suggestion, I think making this `min-height: 54px` would ensure the height will also not shrink.

Reply:
Thanks for the suggestion. 👍